src/Raven.Studio/wwwroot/Content/css/icons.less

```diff
@@ -462,7 +462,6 @@
 .make-icon(running-queries, "\ea1f");
 .make-icon(stats-running-queries, "\ea1f");
 .make-icon(manage-server-running-queries, "\ea1f");
```

Comment:
why icon removed?

Reply:
It had the same code as line 127:
`.make-icon(additional-assemblies, "\ea3a");`
and has been added previously by Rafal with the correct code in line 123:
`.make-icon(spatial-map-view, "\ea40");`